projects/frontend/cicd/install_data_pipelines.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+###
+# This script generates the dependencies tree, installs the packages needed,
+# evaluates the build_data_pipelines_sh script, then
+# lints them to verify format, builds the UI application and runs the unit tests.
+###
+
+if ! which npm >/dev/null 2>&1 ; then
+  echo "ERROR:"
+  echo "Please install npm 8.5.5+. Install cannot continue without it."
+  exit 1
+fi
+echo "Logging npm engines version..."
+npm version
+
+cd "$(dirname $0)" || exit 1
+cd "../data-pipelines/gui"
+
+echo "Removing package lock if available..."
+rm -f "package-lock.json"
+
+shared_dist_dir="../../shared-components/gui/dist/shared"
+if [ -d "$shared_dist_dir" ]
+then
+  echo "Linking the shared-components dist rebuild found..."
+  npm link "$shared_dist_dir"
```

Comment:
This PR depends on having the shared-components build prerequisite, upon shared-components change.
See in `gitlab-ci.yml`:
```frontend_shared_components_changes: &frontend_shared_components_locations```
https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3829441527#L52